src/lib/auth.ts

```diff
@@ -32,6 +32,7 @@ export const ROLES: {
 		'resin.actor.delete?id eq @__ACTOR_ID',
 		'resin.api_key.read?is_of__actor eq @__ACTOR_ID',
 		'resin.application.all',
+		'resin.authentication_delegate.all',
```

Comment:
Right now, a valid user of the API could add/removes/update delegates. This is a PoC and in a real implementation only a user with a security/admin role should be able to modify this.